x-pack/test/api_integration/apis/metrics_ui/log_entry_highlights.ts

```diff
@@ -45,158 +45,171 @@ export default function ({ getService }: FtrProviderContext) {
+                    ? highlightTerms.split(' ')
+                    : [highlightTerms];
+                  expect(column.message[0].highlights).to.eql(expectation);
+                }
+              });
             });
           });
-        });
 
-        it('highlights field columns', async () => {
-          const { body } = await supertest
-            .post(LOG_ENTRIES_HIGHLIGHTS_PATH)
-            .set(COMMON_HEADERS)
-            .send(
-              logEntriesHighlightsRequestRT.encode({
-                logView: { type: 'log-view-reference', logViewId: 'default' },
-                startTimestamp: KEY_BEFORE_START.time,
-                endTimestamp: KEY_AFTER_END.time,
-                highlightTerms: ['generate_test_data/simple_logs'],
-              })
-            )
-            .expect(200);
-
-          const logEntriesHighlightsResponse = pipe(
-            logEntriesHighlightsResponseRT.decode(body),
-            fold(throwErrors(createPlainError), identity)
-          );
-
-          expect(logEntriesHighlightsResponse.data).to.have.length(1);
-
-          const entries = logEntriesHighlightsResponse.data[0].entries;
-
-          // Finds expected entries
-          expect(entries).to.have.length(50);
-
-          // All entries contain the highlights
-          entries.forEach((entry) => {
-            entry.columns.forEach((column) => {
-              if ('field' in column && 'highlights' in column && column.highlights.length > 0) {
-                expect(column.highlights).to.eql(['generate_test_data/simple_logs']);
-              }
+          it('highlights field columns', async () => {
+            const { body } = await supertest
+              .post(LOG_ENTRIES_HIGHLIGHTS_PATH)
+              .set(COMMON_HEADERS)
+              .send(
+                logEntriesHighlightsRequestRT.encode({
+                  logView: { type: 'log-view-reference', logViewId: 'default' },
+                  startTimestamp: KEY_BEFORE_START.time,
+                  endTimestamp: KEY_AFTER_END.time,
+                  highlightTerms: ['generate_test_data/simple_logs'],
+                })
+              )
+              .expect(200);
+
+            const logEntriesHighlightsResponse = pipe(
+              logEntriesHighlightsResponseRT.decode(body),
+              fold(throwErrors(createPlainError), identity)
+            );
+
+            expect(logEntriesHighlightsResponse.data).to.have.length(1);
+
+            const entries = logEntriesHighlightsResponse.data[0].entries;
+
+            // Finds expected entries
+            expect(entries).to.have.length(50);
+
+            // All entries contain the highlights
+            entries.forEach((entry) => {
+              entry.columns.forEach((column) => {
+                if ('field' in column && 'highlights' in column && column.highlights.length > 0) {
+                  expect(column.highlights).to.eql(['generate_test_data/simple_logs']);
+                }
+              });
```

Comment:
I believe the test will look "cleaner" if version is get directly in test:

```
import semver from 'semver';
...
const kibanaServer = getService('kibanaServer');
const currentVersion = kibanaServer.version.get();
const expectation = semver.gte(currentVersion, '8.10.0') ? [highlightTerms] : highlightTerms.split(' ');
```

wdyt?

Reply:
@dmlemeshko thanks for the suggestion. The test is dependent on ES version as there's a change in ES API. Will `kibanaServer.version.get()` work in this case?

Reply:
Hi @dmlemeshko, that inded looks less verbose, but your example bind it to the kibana version and not the ES version. Since our goal here is explicitly to have different behavior on ES compatibility tests this difference is important. Do you know of a way to do what you show in your example but with respect to the ES version?

Reply:
I see. Then you can use `es` service:

```
const es = getService('es');
const response = await es.info();
const version = response.body.version.number;
```